analysis/interface/vtsAnalyser.h

```diff
@@ -114,6 +102,9 @@ class vtsAnalyser : public hadAnalyser {
   float b_axis1,  b_axis2,  b_ptD,       b_area;
   float b_CSVV2;
 
+  float b_dau_pt[100], b_dau_eta[100], b_dau_phi[100];  
+  int   b_dau_charge[100];
```

Comment:
why is this different from the branch size in the Event header?

Reply:
Oops, I forgot to modify that